packages/sku/README.md

```diff
@@ -23,13 +23,17 @@ Create a new project and start a local development environment:
 ```bash
 $ npx sku init my-app
 $ cd my-app
-$ npm start
+$ yarn start
 ```
 
-Don't have [npx](https://www.npmjs.com/package/npx)?
+By default, a new project's dependencies will be installed with the first supported package manager detected on your system.
+Package managers are detected in the following order: `yarn` -> `pnpm` -> `npm`.
+This can be overridden via the `--packageManager` flag:
 
 ```bash
-$ npm install -g npx
+$ pnpm dlx sku init --packageManager pnpm my-app
```

Comment:
```suggestion
$ pnpx sku init --packageManager pnpm my-app
```

Reply:
[`pnpx` doesn't exist anymore](https://github.com/pnpm/pnpm/pull/3652).

Reply:
Or is `pnpx` just an alias for `pnpm dlx` now?

Reply:
I still have it (via Volta), but TIL.